frontend/views/containers/payments/PaymentRowReceived.vue

```diff
@@ -21,7 +21,7 @@
       .cpr-date.has-text-1 {{ humanDate(payment.date) }}
 
     template(slot='cellRelativeTo')
-      .c-relative-to.has-text-1 {{ humanDate(periodStampGivenDate(payment.date)) }}
+      .c-relative-to.has-text-1 {{ humanDate(payment.relativeTo) }}
```

Comment:
Why was this modification made? Is it to support `historicalPeriodStampGivenDate`? If so, see comment below